src/message/InfiniteScrollView.js

```diff
@@ -6,25 +7,47 @@ import config from '../config';
 
 export default class InfiniteScrollView extends React.Component {
 
+  props: {
+    onStartReached?: (() => void) | null,
+    onEndReached?: (() => void) | null,
+    onStartReachedThreshold: number,
+    onEndReachedThreshold: number,
+    onScroll: (e: Event) => void,
+    contentContainerStyle?: Object,
+    style: Object,
+    stickyHeaderIndices: [],
+    autoScrollToBottom: boolean,
+    children?: any[],
```

Comment:
This looks correct, but for built-in React types maybe there is a more concrete and better type already defined?